### ISSUE TITLE
"Send" button is missing sufficient labeling

### ISSUE THREAD

**Issue by cryptasian** [microsoft/BotFramework-WebChat#4988]
### Is it an issue related to Adaptive Cards?

No.

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

ARIA roles, Send box

### What is the public URL for the website?

https://edfinancial.com

### How to reproduce the issue?

This is feedback from the accessibility company Allyant who has been testing our website and has people who use screenreaders and other assistive technologies as a part of their testing team.

![send-button-missing-labeling](https://github.com/microsoft/BotFramework-WebChat/assets/30207740/5bfc936b-9aff-4128-9b06-92d20f8a002f)

### What do you expect?

Add screen reader only text or an aria-label attribute to each button which tells users where it goes and/or what it does.

### What actually happened?

The send button only has a title attribute. The title attribute is not sufficient for button labeling.

### WCAG reference

https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships; https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels
